contrib/gammastep/template.py

```diff
@@ -0,0 +1,30 @@
+pkgname = "gammastep"
+pkgver = "2.0.9"
+pkgrel = 0
+build_style = "gnu_configure"
+make_cmd = "gmake"
+hostmakedepends = [
+    "autoconf",
```

Comment:
```suggestion
```

pulled by autoconf

also sort the *depends lists

Reply:
wdym pulled by autoconf?

Reply:
i meant automake